package.json

```diff
@@ -22,7 +22,7 @@
     "eslint-config-react-app": "^7.0.1"
   },
   "scripts": {
-    "start": "NODE_ENV=development PORT=3001 react-scripts start",
+    "start": "PORT=3001 react-scripts start",
```

Comment:
no longer need the node env?

Reply:
It was confusing the hell out of me so I removed it. It seems to run fine without it.